nameable.rb

```diff
@@ -0,0 +1,6 @@
+# Base class
+class Nameable
+  def correct_name
+    raise NotImplementedError, ' name is not correct'
```

Comment:
- [ ] while raising an error the error shown to user should be specific, Kindly use this one 
- `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`